### ISSUE TITLE
Action failed suddenly

### ISSUE THREAD

**Issue by hyst3ric41** [actions-ecosystem/action-get-latest-tag#27]
Hey there.

I'm using ```actions-ecosystem/action-get-latest-tag@v1``` in my Github Actions workflow and worked as expected.

The step aborted suddenly with no reason apparently and logs shows the following:
![imagen](https://user-images.githubusercontent.com/33683359/200999877-d390d9bd-f2dc-431f-86ed-970d20bf6ff7.png)

Is it an infrequent scenario or it should be receive a patch?

Thanks in advance.

**Comment by thejeff77** [comment]
Still seeing this @josegpulido ?

**Comment by hyst3ric41** [comment]
Yes, but apparently it took place only a twice, and then stopped.

Let me know if I'm an isolated case or other users also were experiecing this.
